protos/flow.proto

```diff
@@ -110,12 +110,9 @@ message TableIdentifier {
   }
 }
 
-message EnsurePullabilityOutput {
-  TableIdentifier table_identifier = 1;
-}
-
 message EnsurePullabilityBatchOutput {
   map<string, TableIdentifier> table_identifier_mapping = 1;
+  MappingType mapping_type = 2;
```

Comment:
Why does the output contain the mapping type?

Reply:
removed